### ISSUE TITLE
[Effect Suggestion] Follow the GPS

### ISSUE THREAD

**Issue by RosyMiranto18** [gta-chaos-mod/ChaosModV#1553]
**Name of effect**
Follow the GPS

**Duration if timed effect (long or short timed?)**
Long

**Effect Description**
When there's a GPS set in a mission or Waypoint you set, you HAVE to follow it exactly during this effect. Deviate from the Route or the Road (including going through the Sidewalk) even for an inch will instantly blow up your car. Watch your map closely.

**Comment by DEATHCON1** [comment]
There would have to be a prompt on screeen when this effect begins that tells you "Follow The GPS" or something like that for to prepare the player for it. Because if not, it will be too sudden...

**Comment by RosyMiranto18** [comment]
Yeah, i think a Prompt of Warning like in Simeon Says is a great idea there. Good touch.
